public/index.html

```diff
@@ -37,15 +37,16 @@
 
                 <div class="content-box">
                     <p class="lead">
-                        Oleme Jaa !mproteater, spontaanne ning lõbus grupp noori amatöörnäitlejaid. Pakume publikule
-                        improviseeritud teatrietendustega rõõmu ja draamat ning õpetame huvilistele improviseerimist.
+                        Jaa !mproteater on Eesti teatrimaastikul tegutsenud aastast 2009 (enne 2016 nimega Improgrupp Jaa!). 
```

Comment:
On formatting: would it look prettier with some paragraph breaks and perhaps with `<p class="text-justify">` (justify alignment)?